compose/install/index.md

```diff
@@ -2,31 +2,45 @@
 description: How to install Docker Compose
 keywords: compose, orchestration, install, installation, docker, documentation
 title: Install Docker Compose
-toc_max: 2
+toc_max: 3
+redirect_from:
+- /compose/compose-desktop/
 ---
 
-On this page you can find a summary of the available options for installing Docker Compose. 
+On this page you can find information on how to get and install Compose.
 
-## Compose prerequisites
+## Install Compose
 
-* Docker Compose requires Docker Engine.
-* Docker Compose plugin requires Docker CLI.
+If you have Docker Desktop, you've got a full Docker installation, including Compose.
 
-## Compose installation scenarios
-You can run Compose on macOS, Windows, and 64-bit Linux. Check what installation scenario fits your needs. 
+You can check this by clicking on **About Docker Desktop** from the Docker menu ![whale menu](../../desktop/images/whale-x.png){: .inline}.
 
-Are you looking to:
+![about-desktop](../images/about_desktop.png){:width="750px"}
 
-* __Get latest Docker Compose and its prerequisites__:
-[Install Docker Desktop for your platform](./compose-desktop.md). This is the fastest route and you get Docker Engine and Docker CLI with the Compose plugin. Docker Desktop is available for Mac, Windows and Linux.
+### New Docker Installation
+
+If you haven’t yet installed Docker Desktop:
+
+{% assign yes = '![yes](/images/green-check.svg){: .inline style="height: 14px; margin: 0 auto; align=right"}' %}
+
+| Platform       | Docker Desktop                 |
+|:---------------|:------------------------------:|
+|Linux (64-bit)  |{{ yes }} [Install](../../desktop/install/linux-install.md)|
+|Mac             |{{ yes }} [Install](../../desktop/install/mac-install.md)|
+|Windows         |{{ yes }} [Install](../../desktop/install/windows-install.md)|
+
+
+### Pre-existing Docker Installation
+
+If you have a Docker installation on Linux to which you want to add the Compose plugin, you can choose one of these options:
+
+* [Install using the repository](./compose-linux.md#install-using-the-repository)
+* [Install the plugin manually](./compose-linux.md#install-the-plugin-manually)
+
+### Other install scenarios
+
+For other options check, [Other install scenarios](./compose-other.md/).
```

Comment:
The "check," looks like a typo. Also, suggestion for a less redundant phrasing:

```suggestion
See [Other Install Scenarios](./compose-other.md/).
```

Reply:
Adding see after the comma.

Reply:
Resolving addressed comment.